PySDM/physics/isotope_kinetic_fractionation_factors/pierchala_et_al_2022.py

```diff
@@ -0,0 +1,15 @@
+"""
+kinetic fractionation factor and epsilon kinetic
+from [Pierchala et al. 2022](https://doi.org/10.1016/j.gca.2022.01.020)
+"""
+
+
+class PierchalaEtAl2022:  # pylint: disable=too-few-public-methods
+    @staticmethod
+    def alpha_kinetic(eps_kinetic):
+        return 1 - eps_kinetic
+
+    @staticmethod
+    def eps_kinetic(*, theta, n, eps_diff, relative_humidity):
```

Comment:
let's then remove it from the Pierchala notebook and use `formulae.isotope_kinetic_fractionation_factors.eps_kinetic` there